safe_web_api_playground/static/js/code-snippets/safeImmutableData.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
       } catch(err) {
         return err;
       }
-      return `Returns ImmutableData data: ${String.fromCharCode.apply(null, new Uint8Array(data))}`;
+      return `Returns ImmutableData data: ${String.fromCharCode.apply(null, data)}`;
```

Comment:
This still needs to be converted to `Uint8Array` as it's being returned as `ArrayBuffer`

Reply:
I'm seeing it returned as Uint8Array:
![returns_uint8array](https://user-images.githubusercontent.com/10050403/35285486-5a0b2ac4-0012-11e8-84e2-e6d15c40dabd.PNG)